_includes/pbc/all/install-features/202304.0/install-the-gift-cards-feature.md

```diff
@@ -0,0 +1,1338 @@
+| spy_gift_card_product_configuration_link | table | created |
+| spy_payment_gift_card | table | created |
+| spy_gift_card_balance_log | table | created |
+| spy_sales_order_item_gift_card | table | created |
+
+Make sure that propel entities have been generated successfully by checking their existence. Also, change the generated entity classes to extend from Spryker core classes.
+
+| CLASS PATH | EXTENDS |
+| --- | --- |
+| src/Orm/Zed/GiftCard/Persistence/SpyGiftCard.php | Spryker\\Zed\\GiftCard\\Persistence\\Propel\\AbstractSpyGiftCard |
+| src/Orm/Zed/GiftCard/Persistence/SpyGiftCardQuery.php | Spryker\\Zed\\GiftCard\\Persistence\\Propel\\AbstractSpyGiftCardQuery |
+| src/Orm/Zed/GiftCard/Persistence/SpyGiftCardProductAbstractConfiguration.php | Spryker\\Zed\\GiftCard\\Persistence\\Propel\\AbstractSpyGiftCardProductAbstractConfiguration |
+| src/Orm/Zed/GiftCard/Persistence/SpyGiftCardProductAbstractConfigurationQuery.php | Spryker\\Zed\\GiftCard\\Persistence\\Propel\\AbstractSpyGiftCardProductAbstractConfigurationQuery |
+| src/Orm/Zed/GiftCard/Persistence/SpyGiftCardProductAbstractConfigurationLink.php | Spryker\\Zed\\GiftCard\\Persistence\\Propel\\AbstractSpyGiftCardProductAbstractConfigurationLink |
+| src/Orm/Zed/GiftCard/Persistence/SpyGiftCardProductAbstractConfigurationLinkQuery.php | Spryker\\Zed\\GiftCard\\Persistence\\Propel\\AbstractSpyGiftCardProductAbstractConfigurationLink |
+| src/Orm/Zed/GiftCard/Persistence/SpyGiftCardProductConfiguration.php | Spryker\\Zed\\GiftCard\\Persistence\\Propel\\AbstractSpyGiftCardProductConfiguration |
+| src/Orm/Zed/GiftCard/Persistence/SpyGiftCardProductConfigurationQuery.php | Spryker\\Zed\\GiftCard\\Persistence\\Propel\\AbstractSpyGiftCardProductConfigurationQuery |
+| src/Orm/Zed/GiftCard/Persistence/SpyGiftCardProductConfigurationLink.php | Spryker\\Zed\\GiftCard\\Persistence\\Propel\\AbstractSpyGiftCardProductConfigurationLink |
+| src/Orm/Zed/GiftCard/Persistence/SpyGiftCardProductConfigurationLinkQuery.php | Spryker\\Zed\\GiftCard\\Persistence\\Propel\\AbstractSpyGiftCardProductConfigurationLinkQuery |
+| src/Orm/Zed/GiftCard/Persistence/SpyPaymentGiftCard.php | Spryker\\Zed\\GiftCard\\Persistence\\Propel\\AbstractSpyPaymentGiftCard |
+| src/Orm/Zed/GiftCard/Persistence/SpyPaymentGiftCardQuery.php | Spryker\\Zed\\GiftCard\\Persistence\\Propel\\AbstractSpyPaymentGiftCardQuery |
+| src/Orm/Zed/GiftCardBalance/Persistence/SpyGiftCardBalanceLog.php | Orm\\Zed\\GiftCardBalance\\Persistence\\Base\\SpyGiftCardBalanceLog |
+| src/Orm/Zed/GiftCardBalance/Persistence/SpyGiftCardBalanceLogQuery.php | Orm\\Zed\\GiftCardBalance\\Persistence\\Base\\SpyGiftCardBalanceLogQuery |
+| src/Orm/Zed/Sales/Persistence/SpySalesOrderItemGiftCard.php | Spryker\\Zed\\Sales\\Persistence\\Propel\\AbstractSpySalesOrderItemGiftCard |
+| Spryker\\Zed\\Sales\\Persistence\\Propel\\AbstractSpySalesOrderItemGiftCardQuery |
+
+{% endinfo_block %}
+
+### 4) Import data
```

Comment:
This section is blank and does not provide any extra information, therefore we can remove it and replace it with the subsection. 
```suggestion
### 5) Import the Gift Card configuration data
```